templates_cspm_eventbridge/OrgFullInstall.yaml

```diff
@@ -161,7 +157,7 @@ Resources:
       StackInstancesGroup:
         - DeploymentTargets:
             OrganizationalUnitIds: !Split [ ",", !Ref OrganizationUnitIDs]
-          Regions: !Split [ ",", !Ref Regions]
+          Regions: [!Ref "AWS::Region"]
```

Comment:
@nkraemer-sysdig need to test if this works.

Reply:
Already tested: Full install stack here: https://us-east-2.console.aws.amazon.com/cloudformation/home?region=us-east-2#/stacks/parameters?stackId=arn%3Aaws%3Acloudformation%3Aus-east-2%3A237944556329%3Astack%2Fnoah%2F5d7dfc70-d967-11ed-82a2-0a41e7f00a87&filteringText=&filteringStatus=active&viewNested=true